src/components/ProjectHeader.jsx

```diff
@@ -16,12 +16,17 @@ class ProjectHeader extends React.Component {
     this.toggleDropdown = this.toggleDropdown.bind(this);
   }
 
+  componentWillReceiveProps(next) {
+    if (this.props.location.pathname !== next.location.pathname && this.props.showWorkflows) {
```

Comment:
I like that we have a bit of code here that closes the Classify->Workflow Selection dropdown when we change pages, though we should get into a better practice of noting down what's happening with a short comment.